src/manage-apps/using-terminal.md

```diff
@@ -0,0 +1,114 @@
+# Installing Applications with the Terminal
+
+The Terminal provides more flexibility and efficiency when installing applications. Apt and Flatpak are powerful package management tools that allow users to install, update, and remove packages using simple and intuitive commands.
+
+---
+
+## Understanding Package Managers
+
+A package manager is an application that keeps track of packages' files on your computer. A package manager can also verify and retrieve dependencies for any program the user wishes to install.
+
+### The Advanced Packaging Tool (Apt)
+
+Pop!\_OS comes preinstalled with the Advanced Packaging Tool (`apt`). `Apt` is a package manager that lies on top of another package manager called `dpkg`.  `Apt` accomplishes this by referencing its own database of packages, called a repository.
```

Comment:
> `Apt` accomplishes this by referencing its own database of packages, called a repository.

It accomplishes what? You haven't said anything about what `apt` accomplishes in this paragraph, only that it lies on top of `dpkg`.

Also, you might want to be careful about capitalizing `a` and `d` in these names when you're putting them in code backticks. The commands are case-sensitive.

Reply:
I've added a table to better clarify the functionality and use-cases for `apt` and `dpkg`. I'm sure that I've missed something, so please let me know what should be added.